### ISSUE TITLE
Rename "Fees on top" to "Platform tip" in the code

### ISSUE THREAD

**Issue by Betree** [opencollective/opencollective#5049]
While we're using "Platform tip" everywhere in the interface and in the docs, many places of the code still use "feesOnTop". We should rename them. Amon other things:
- Variables in the frontend
- Variables in the api
- `order.data.isFeesOnTop` flag

**Comment by kewitz** [comment]
AFAIK we also don't charge platform fees anymore so we should also deprecate the `data.platformFee`.
Also, I suggest we move the tip information to a proper column in the Orders table.

**Comment by SudharakaP** [comment]
Yes, as just FYI; maybe related to this, I've opened the following issue https://github.com/opencollective/opencollective/issues/5052 and three PRs to do the migration related to the `platformContributionAmount`. Ideally I think we should not have any `platformContributionAmount` but should use only `platformTipAmount` in API. 🤔

**Comment by Betree** [comment]
Cool, let's be careful about Metabase queries while doing these updates. There are probably some which depend on the current flag names.
